NBXplorer.Client/NBXplorerNetworkProvider.Liquid.cs

```
@@ -29,6 +29,11 @@ public override BitcoinAddress CreateAddress(DerivationStrategyBase derivationSt
 
 			public static Key GenerateBlindingKey(DerivationStrategyBase derivationStrategy, KeyPath keyPath)
```

this should throw `InvalidOperationException` if it is unblinded.

---

fixed